### PR TITLE
Dev GUI

### PR DESCRIPTION
**Pull request by Ekkiddle**
Since this is getting big, we might want to do some merging...

### CODE REVIEW

File: com.thelocalmarketplace.software/src/com/thelocalmarketplace/software/SelfCheckoutStationLogic.java
Comment (CeeRobitaille) [inline com.thelocalmarketplace.software/src/com/thelocalmarketplace/software/SelfCheckoutStationLogic.java:1]: Changes to Logic are just cosmetic, they wont break anything

**Review by CeeRobitaille — CHANGES_REQUESTED** [review]
The only issues that could cause issues are in the testing package. Please merge main onto your branch to get the more resent test package structure or else this will cause issues in the future.

**Review by CeeRobitaille — APPROVED** [review]
Tests have been fixed. No noteworthy changes outside of GUI stuff, so shouldnt impact other parts of the project.

**Review by kingsleyzhong — APPROVED** [review]
Looks great!

**Pull request merged**